docs/docusaurus/src/css/navbar.scss

```diff
@@ -86,6 +93,11 @@
 
 .navbar__link--active {
   border-bottom: 2px solid var(--ifm-color-primary);
+
+  &[href*="0.17"] {
```

Comment:
This is added to prevent all three tabs (GX OSS, learn and api) from being selected at the samte in version 0.17. The source problem relies on a retrocompatibility problem due to extensive changes in docs structure from version 0.17 to 0.18 which will be solved by cutting this version out of the current project (similar to what has been done with version 0.16)

Reply:
Note to anyone else looking at this: I'm fine with this approach because it's an improvement, and archiving 0.17 later this week will make fully resolve the issue.